.gitignore

```diff
@@ -3,9 +3,6 @@ __pycache__/
 *.py[cod]
 *$py.class
 
-# C extensions
-*.so
-
```

Comment:
This might make a merge conflict with your other pull request:
https://github.com/cha2251/rit-player-piano/pull/10/files

Reply:
See comment in PR 10. Pretty sure this will get auto resolved